addons/website_slides/static/tests/tours/slides_course_member.js

```diff
@@ -156,7 +156,24 @@ registry.category("web_tour.tours").add("course_member", {
         },
         {
             trigger: ".modal.modal_shown .modal-body textarea",
-            run: "edit This is a great course. Top !",
+            run: "edit This is a great course. Top!",
+        },
+        {
+            trigger: ".modal.modal_shown button:contains(review)",
+            run: "click",
+        },
+        // eLearning: edit the review
+        {
+            trigger: 'button[data-bs-target="#ratingpopupcomposer"]:contains("Edit Review")',
+            run: "click",
+        },
+        {
+            trigger: ".modal.modal_shown .modal-body i.fa.fa-star-o:eq(1)",
+            run: "click",
```

Comment:
It's this step right?

If it's just a "check" step to make sure the previous action succeeded before going further, maybe you don't need to click on it? I seem to remember a specific attribute on steps for that, "isCheck: true" or something, but I'm not 100% sure :thinking:

Reply:
It's not really a check step. It changes the rating value from 3 to 5, and this makes sense because the user can change the text and rating both.